siemens_to_ismrmrd/meta.yaml

```diff
@@ -24,4 +26,4 @@ requirements:
     - libxslt=1.1.33 
 
 about:
-  home:
\ No newline at end of file
+  home:
```

Comment:
Let's take the opportunity to add the metadata here as we edit this file anyway.